src/ray/gcs/gcs_server/gcs_node_manager.h

```diff
@@ -167,6 +173,8 @@ class GcsNodeManager : public rpc::NodeInfoHandler {
   std::shared_ptr<gcs::GcsTableStorage> gcs_table_storage_;
   /// Raylet client pool.
   std::shared_ptr<rpc::NodeManagerClientPool> raylet_client_pool_;
+  /// Cluster ID.
+  const ClusterID &cluster_id_;
```

Comment:
nit: can we make it a copy?

Reply:
https://github.com/ray-project/ray/pull/36738

Reply:
Could you elaborate it?

Reply:
Could you elaborate it?

Reply:
I'd also prefer a copy if possible: because then it's clear to me that the cluster id will not be changed.